code/index.html

```diff
@@ -2,29 +2,209 @@
+      </div>
+
+
+    </div>
+    
+    <!---Sticky navbar-->
+    <header class="header">
+
+      <nav>
+        <a href="#">ABOUT</a>
+        <a href="#">READ</a>
+        <a href="#">ARCHIVE</a>
+        <a href="#">SUBSCRIBE</a>
+        <a href="#">LOG IN</a>
+      </nav>
+
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+    </header>
+
+    <!--extra content-->
+    <section class="other">
+      <div class="information">
+        <h3> This is a text about yoga</h3>
+        <p>Our life today is full of rush, stress and irritability. By applying the universal principles of yoga you can gain mental balance, better sleep and a new vitality. Meditation improves your concentration and enriches your life with a new creativity.
+
+          You are welcome to visit on your own, with a friend or bring the entire family. Enjoy the fresh air, green meadows, quiet forests, medicinal plants along scenic foot-paths, sparkling brooks and pristine lakes.
+          
+          Everybody in our staff team is dedicated to a practical yoga life style, and we are looking forward to being of service to you during your stay.
+           
+          Swami Durgananda, director of the Sivananda Yoga Centres in Europe</p>
+
+      </div>
+
+      <div class="picture">
+        <img src="videos/bigpic.jpg" alt="Girl doing yoga">
+      </div>
+
+    </section>
+
+    <!--Main Content  (for class) -->
+    <section class="maincontent">
+
+    <!--  Form -->
+   <section class="form">
+
+        <!--Sign up -->
+   <div class="signup">
+    <h2>Yoga with us</h2>
+
+    <!--Textbox-->
+    <form action="http://httpbin.org/anything" method="POST">
+     <input type="name" name="name" placeholder="First name"> 
+     <input type="name" name="name" placeholder="Last name">>
+<br>
+<!--Scroll-select-->
+<label for="age">How old are you?</label>
+   <select name="Age">
+     <option value="10">10+</option>
+     <option value="20">20+</option>
+     <option value="30">30+</option>
+     <option value="40">40+</option>
+     <option value="50">50+</option> 
+     <option value="60">60+</option>
+     <option value="70">70+</option> 
+     <option value="80">80+</option>
+     <option value="90">90+</option> 
+   </select>
+   <br>
+
+   <!--Radio-->
+    <p>What type of yoga are you interested in?:</p>
+    <input type="radio" name="yoga" value="Ashtanga" multiple>
+    <label for="Ashtanga">Ashtanga</label>
+    <br>
+    <input type="radio" name="yoga" value="Vinyasa">
+    <label for="Vinyasa">Vinyasa</label>
+    <br>
+    <input type="radio" name="yoga" value="Yin">
+    <label for="Yin">Yin</label>
+    <br>
+    <input type="radio" name="yoga" value="Bikram">
+    <label for="Bikram">Bikram</label>
+    <br>
+    <input type="radio" name="yoga" value="Hatha">
+    <label for="Hatha">Hatha</label>
+    <br>
```

Comment:
Please structure your indentation, this is hard to read. Readable and structured code is SO important for yourself and also future employer, right?
If you're gonna connect the label with the input by using the 'for'-attribute then it should be connected by the id, here's some reading:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label